config/systemjs.config.js

```diff
@@ -90,13 +122,18 @@ System.config({
       "fs": "github:jspm/nodelibs-fs@0.1.2",
       "inherits": "npm:inherits@2.0.1",
       "minimalistic-assert": "npm:minimalistic-assert@1.0.0",
+      "systemjs-json": "github:systemjs/plugin-json@0.1.0",
```

Comment:
Doing a clean install, the only differences in mine are:

1. Upgrades rxjs to `beta.3` (which we don't want)
2. Removes the top-level zone.js dependency (but leaves the nested ones)